### ISSUE TITLE
Add functionality to check an API key is valid

### ISSUE THREAD

**Issue by IanHopkinson** [OCHA-DAP/hdx-cli-toolkit#30]
Add functionality to the `configuration` command to check whether a supplied token is valid for the operation requested.

**Comment by IanHopkinson** [comment]
@mcarans has implemented a function to check API keys here:
```
if not User.check_current_user_organization_access("hdx", "create_dataset"):
    raise PermissionError(
        "API Token does not give access to HDX organisation!"
    )
```

**Comment by IanHopkinson** [comment]
Fixed in #32 v2024.6.1
